qutebrowser/commands/command.py

```diff
@@ -170,6 +171,8 @@ def get_arg_info(self, param):
 
     def get_pos_arg_info(self, pos):
         """Get an ArgInfo tuple for the given positional parameter."""
+        if pos >= len(self.pos_args) and self._has_vararg:
+            pos = len(self.pos_args) - 1
```

Comment:
A bit of a nitpick, but this can be:

```
if self._has_vararg:
  pos = min(len(self.pos_args) - 1, pos))
```

Reply:
I think the current way is a little more readable

Reply:
Just to join the nitpicking, my preferred version would be

```python
if self._has_vararg  and pos >= len(self.pos_args):
    pos = len(self.pos_args) - 1
```
:wink: